gulpfile.js

```diff
@@ -794,11 +794,14 @@ function parseExtensionFlags() {
       if (argv.extensions === 'minimal_set') {
         argv.extensions = MINIMAL_EXTENSION_SET.join(',');
       }
+    }
 
+    if (argv.extensions || argv.extensions_from) {
+      const extensionsToBuild = getExtensionsToBuild();
```

Comment:
I don't see where the value of `extensionsToBuild` is assigned back to `argv.extensions` here. Can you check?

Reply:
Same here. We do this because `extensionsToBuild` is not global. And we need to append the extension to `argv.extensions` to make it global. Again I'm happy to fix this with following change.

Reply:
Ideally, we should call `getExtensionsToBuild` once at the start, and use its result all over the place. Currently, we write the computed list of extensions back to `argv.extensions`, but I'm fine with writing it to `extensionsToBuild` and making it global to this file. Your call!

Reply:
made `extensionsToBuild` global.